gemfirexd/tools/src/test/java/com/pivotal/gemfirexd/jdbc/LangScripts_LOBTest.java

```diff
@@ -597,10 +597,10 @@ public void testLangScript_LOB_WithPartitioning() throws Exception {
       { "create table e(a string(204K)) partition by column(a)"+ getOffHeapSuffix(), null },
       // Create index (not allowed)
       { "create index ia on a(a)", "X0X67" },
-      { "create index ib on b(a)", null },
+      { "create index ib on b(a)", "42832" },
       { "create index ic on c(a)", "42Y55" },
-      { "create index id on d(a)", null },
-      { "create index ie on e(a)", null },
+      { "create index id on d(a)", "42832" },
+      { "create index ie on e(a)", "42832" },
```

Comment:
LOB as PK or index key got explicitly disabled with the unified partitioner merge which had my patch for the same. Allowing this had problems because we have no handling of *CompositeKeys for byte[][]'s especially if the key is a combination of LOB and non-LOB (then it needs to pick up a combo of LOB byte[] and first byte[]).